src/services/ProfileApiService.js

```diff
@@ -37,7 +37,16 @@ export async function patchProfile(username, params) {
         'Content-Type': 'application/merge-patch+json',
       },
     },
-  );
+  ).catch((error) => {
+    const processedError = Object.create(error);
+    const fieldErrors = Object.entries(processAccountData(error.response.data.field_errors))
+      .reduce((acc, [fieldKey, messages]) => {
+        acc[fieldKey] = messages.userMessage;
+        return acc;
+      }, {});
+    processedError.fieldErrors = fieldErrors;
```

Comment:
Might just call this "errors" since that's what it's becoming a step or two later anyway... that way the name just stays consistent.  And it's shorter that way. ;)

Reply:
fieldErrors is clearer, no? I actually think maybe field errors is a better name throughout.